src/main/frontend/pipeline-console-view/pipeline-console/main/ConsoleLogStream.tsx

```diff
@@ -73,6 +73,21 @@ export default function ConsoleLogStream(props: ConsoleLogStreamProps) {
     return props.step.state === Result.running || props.logBuffer.startByte < 0;
   };
 
+  useEffect(() => {
+    const hash = window.location.hash;
+    if (!hash.startsWith("#log-")) {
+      return;
+    }
+    const lineNumber = parseInt(hash.substring(5));
+    if (!(!isNaN(lineNumber) && virtuosoRef.current)) {
+      return;
+    }
+    virtuosoRef.current.scrollToIndex({
+      index: lineNumber,
+      align: "center",
+    });
```

Comment:
I tried setting the [behaviour](https://virtuoso.dev/virtuoso-api/interfaces/FlatIndexLocationWithAlign/#behavior) property to `smooth` but then it wouldn't do anything